hp_list/Makefile

```diff
@@ -12,6 +12,17 @@ $(BIN): main.c
 all: CFLAGS += -O2
 all: $(BIN)
 
+# The RUNTIME_STAT allow to show the runtime operation states
+# in the list_insert, list_delete and __list_find function by
```

Comment:
You can delete the comments starting from "in the list_insert, list_delete and ..." since they were mentioned in C source file.